### ISSUE TITLE
remove references to the "gallery" that does not exist

### ISSUE THREAD

**Issue by kolibril13** [napari/napari#3531]
Here is mentioned a gallery: https://napari.org/tutorials/index.html#getting-started
![image](https://user-images.githubusercontent.com/44469195/135449102-20eb34d9-09e6-4627-a204-73b45bfa467b.png)
but it is not clear where to find it.
Can one add here a link to the gallery?

**Comment by GenevieveBuckley** [comment]
Next steps: we need to remove all references to a napari gallery (check both napari/napari & napari/napari.github.io), since a gallery does not exist yet. This would be a good first issue, if anyone is interested.

**Comment by GenevieveBuckley** [comment]
I started to make a PR for this today, but I don't think it's actually an issue anymore? Given that, I'm going to close the issue.

I can't find any references to "gallery" in either the napari/napari or napari/napari.github.io repositories (besides [this one I have made a comment about](https://github.com/napari/docs/issues/39) and we will follow up on that other issue)

Searches for "gallery" in napari repositories:
- https://github.com/napari/napari.github.io/search?q=gallery
- https://github.com/napari/napari/search?q=gallery

I don't know which PRs might have changed this, there's nothing linked back to this original issue.
